core/src/main/java/org/mule/runtime/core/api/scheduler/SchedulerConfig.java

```diff
@@ -63,4 +85,27 @@ public SchedulerConfig withName(String schedulerName) {
   public String getSchedulerName() {
     return schedulerName;
   }
+
+  /**
+   * Sets the rejection policy to use when dispatching to a busy {@link Scheduler}.
+   * <p>
+   * This is only applicable for <b>custom</b> {@link Scheduler}s. The policy cannot be changed for the runtime managed
+   * {@link Scheduler}.
+   * 
+   * @see SchedulerBusyException
+   * 
+   * @return the updated configuration
+   */
+  public SchedulerConfig withDispatchingRejectionPolicy(DispatchingRejectionPolicy dispatchingRejectionPolicy) {
```

Comment:
mmm, that may bring some confusion with the rejectionhandler of an executor. i wanted to give it a name that clearly differentiates from it.